src/modules/autoresponses/textResponses.js

```diff
@@ -15,10 +15,7 @@ export async function messageCreate(client, message) {
             const triggers = responseDto.trigger.some((trigger) =>
                 trigger instanceof RegExp
                     ? message.content.toLowerCase().match(trigger)
-                    : message.content
-                          .toLowerCase()
-                          .split(' ')
-                          .some((w) => w === trigger)
+                    : message.content.toLowerCase().match(new RegExp(`\\b${trigger}\\b`))
```

Comment:
Do we want to filter out any regex data in here? As these are not regex triggers its assumed we don't want to trigger regex in them. A user might put something such as a ^ and now its trying to check if its the begining.

Reply:
See: https://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex

Reply:
A valid point but what are the chances if this ever happening?